src/pages/CourseSelector/CourseList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Rather than creating a course list, you should create a `course tag` component which can be reused. This way, it's more generic and can be customised to be used in other pages as well.

For better logic, place the `course tag` inside components instead of `courseSelector`